<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="EPINHYDW022A" failures="1" tests="3" name="REgTEst" time="0.03" errors="0" timestamp="2022-01-31T14:03:01 IST">
  <testcase classname="com.practice.test.Rough1" name="doLogin" time="0.008"/>
  <testcase classname="com.practice.test.Rough1" name="doReg" time="0.002">
    <failure type="java.lang.AssertionError" message="Test Passed expected [Hie] but found [Hi]">
      <![CDATA[java.lang.AssertionError: Test Passed expected [Hie] but found [Hi]
at com.practice.test.Rough1.doReg(Rough1.java:15)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- doReg -->
  <testcase classname="com.practice.test.Rough1" name="doAccountCreatn" time="0.001"/>
</testsuite> <!-- REgTEst -->
